<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>331</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CAM1</string>
     </attribute>
     <widget class="QLabel" name="titleSensorMode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sensor Mode:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sensorModespinBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>291</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSlider" name="FocusSlider">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>122</x>
         <y>11</y>
         <width>78</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Âµs</string>
       </property>
      </widget>
      <widget class="QSlider" name="ExposureTimeSlider">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="labelTitleExposureTime">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>105</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Exposure Time:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelGain">
       <property name="geometry">
        <rect>
         <x>109</x>
         <y>61</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ns</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFrameDuration">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ns</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleFocus">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>111</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Frame Duration:</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleGain">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>61</y>
         <width>92</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Gain:</string>
       </property>
      </widget>
      <widget class="QSlider" name="GainSlider">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="labelCurrFrameRate">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelResolution">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleResolution">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Resolution: </string>
       </property>
      </widget>
      <widget class="QLabel" name="titleFrameRate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Avg Frame Rate:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFrameRate">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleCurrFrameRate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Current FR: </string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>540</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Data</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioThreshold">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioFloodFill">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>FloodFill</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioGray">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Gray Scale</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioOriginal">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Original</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sensorModeApplyButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="main_stopButtonCAM1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="main_startButtonCAM1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>CAM2</string>
     </attribute>
     <widget class="QRadioButton" name="radioGray_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Gray Scale</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="labelCurrFrameRate_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelResolution_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleResolution_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Resolution: </string>
       </property>
      </widget>
      <widget class="QLabel" name="titleFrameRate_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Avg Frame Rate:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFrameRate_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleCurrFrameRate_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Current FR: </string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="sensorModeApplyButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>291</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSlider" name="FocusSlider_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>122</x>
         <y>11</y>
         <width>78</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Âµs</string>
       </property>
      </widget>
      <widget class="QSlider" name="ExposureTimeSlider_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="labelTitleExposureTime_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>105</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Exposure Time:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelGain_2">
       <property name="geometry">
        <rect>
         <x>109</x>
         <y>61</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ns</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFrameDuration_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ns</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleFocus_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>111</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Frame Duration:</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleGain_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>61</y>
         <width>92</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Gain:</string>
       </property>
      </widget>
      <widget class="QSlider" name="GainSlider_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QRadioButton" name="radioFloodFill_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>FloodFill</string>
      </property>
     </widget>
     <widget class="QLabel" name="titleSensorMode_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sensor Mode:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sensorModespinBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioOriginal_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Original</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioThreshold_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QPushButton" name="main_stopButtonCAM2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="main_startButtonCAM2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Colour</string>
     </attribute>
     <widget class="QPushButton" name="colourInitButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Colour</string>
      </property>
     </widget>
     <widget class="QLabel" name="titleColour">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Colour: </string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>151</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelColourL">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelColourA">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelColourB">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="colourDisplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="titleColour_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera 1: </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>361</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="QimageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string>Live Label</string>
     </property>
    </widget>
    <widget class="QLabel" name="Qimage2Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CAMERA 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CAMERA 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>291</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="QimageDefect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>Live Label</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>465</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="captureButton">
       <property name="text">
        <string>Capture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="triggerModeButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
